packages/atomic/src/components/recommendations/atomic-recs-interface/atomic-recs-interface.tsx

```diff
@@ -203,6 +205,13 @@ export class AtomicRecsInterface
     );
   }
 
+  @Method() public async getOrganizationEndpoints(
```

Comment:
I think we need this for the atomic-search-interface docs:
```suggestion
  /**
   * Returns the unique, organization-specific endpoint(s) using options for `orgId` (required) and `env` (optional — possible values:`prod`, `hipaa`, `staging`,`dev`. Defaults to `prod`).
   */
  @Method() public async getOrganizationEndpoints(
```

Reply:
Also, I think we need to modify line 157/159 for `initialize`:

Initializes the connection with the headless search engine using options for `accessToken` (required), `organizationId` (required), `renewAccessToken`, `organizationEndpoints` (recommended), and `platformUrl` (deprecated).